Makefile

```diff
@@ -111,10 +111,10 @@ run-integ-test:
 	go test -race -count=1 -timeout 120m -tags=integration ${PACKAGES}
 
 .PHONY: local-test
-local-test: package-custom-resources custom-resource-tests run-local-test package-custom-resources-clean
+local-test: package-custom-resources custom-resource-tests run-local-integ-test run-unit-test package-custom-resources-clean
 
 .PHONY: run-local-test
-run-local-test:
+run-local-integ-test:
```

Comment:
don't the unit tests already get run here? so now this'll just run them twice?

Reply:
Didn't realize `--tag` was inclusive and was not excluding tests that are not tagged.

Anyway, I ended up with this PR because I found out that `make test` was not passing at `StaticSite_Parameters` test, and the fact that `make local-test` was passing made me think that it wasn't running unit test.

The real reason turns out to be that `StaticSite_Parameters` didn't stub the `fs`, therefore it required `package-custom-resources` to pass.

I've fixed it accordingly.